### PR TITLE
Port Scan - Generic

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
- [ ] In Progress
- [x] Ready
- [ ] On Hold - should merge with this https://github.com/demisto/content/pull/6285

## Related Issues
fixes: https://github.com/demisto/etc/issues/23152
fixes: https://github.com/demisto/etc/issues/23153

## Description
- Added a generic port scan use case.
Includes 3 playbooks, a layout, fields and an incident type.
- Added readme

## Screenshots

image

Port_Scan_-_Generic_Tue_Apr_07_2020

Port_Scan_-_Internal_Source_dev_Tue_Apr_07_2020

Port_Scan_-_External_Source_dev_Tue_Apr_07_2020

## Minimum version of Demisto
5.0.0

## Does it break backward compatibility?
No

### CODE REVIEW

**Comment by yaron-libman** [comment]
Reviewed and approved.

File: Packs/PortScan/Playbooks/Port_Scan_-_Internal_Source.yml [hunk 1]
```diff
@@ -0,0 +1,3833 @@
...
+  '118':
+    id: '118'
+    taskid: 205a206f-0abd-4863-8110-30f4a80230ed
+    type: regular
+    task:
+      id: 205a206f-0abd-4863-8110-30f4a80230ed
+      version: -1
+      name: Save user info and manager email in incident fields
+      description: Saves the user's email address and display name to the Employee Email and Employee Display Name fields, respectively,
+      and the user's manager email is saved to the Employee Manager Email field.
```
Comment (amshamah419) [inline Packs/PortScan/Playbooks/Port_Scan_-_Internal_Source.yml:1553]:
```suggestion
      description: "Saves the user's email address and display name to the Employee Email and Employee Display Name fields, respectively,\
      \and the user's manager email is saved to the Employee Manager Email field."
```

Comment (amshamah419) [reply Packs/PortScan/Playbooks/Port_Scan_-_Internal_Source.yml:1553]: Try this. I think the commas are not being handled correctly and the yml validation fails

**Comment by idovandijk** [comment]
Fails due to ExtraHop test that failed. Our license expired. Now waiting to see if we can get a new license, otherwise I will skip their test.

**Comment by idovandijk** [comment]
Skipped ExtraHop because of lack of license

**Pull request merged**